### PR TITLE
Edit Subscribe Interactions

### PR DESCRIPTION
**Pull request by fluffy**

### CODE REVIEW

**Review by suhasHere — APPROVED** [review]

**Review by fluffy — COMMENTED** [review]
LGTM

**Review by afrind — COMMENTED** [review]
LGTM

File: draft-lcurley-moq-transport.md [hunk 1]
```diff
@@ -470,15 +470,13 @@ and publish requests to the tracks.
...
+The endpoint making the subscribe request is notified of the result of the subscription, via "SUBSCRIBE OK" ({{message-subscribe-ok}}) or the "SUBSCRIBE ERROR" {{message-subscribe-error}} control message.
 
-For successful subscriptions, relays proceed to save the subscription information by maintaining mapping from the track information to the list of subscribers. This will enable relays to forward matching publishes on the requested track. Subscriptions stay active until it is expired or the publisher of the track stops producing objects or other reasons that result in error (see {{message-subscribe-error}}).
+For successful subscriptions, the sender maintains a list of subscribers for each full track name. Each new OBJECT belonging to the track MUST be forwarded to each active subscriber, unless determined by congestion response. A subscription remains active until it expires, or until the publisher of the track stops producing objects or there is a subscription error (see {{message-subscribe-error}}).
```
[the remark this answers is not in the document]
Comment (kixelated) [reply draft-lcurley-moq-transport.md:477]: > A subscription remains active until it expires, or until the publisher of the track stops producing objects or there is a subscription error

suggested replacement:

> A subscription remains active until the publisher sends a SUBSCRIBE ERROR message.

We could add some examples but I think it's fine without them.

**Review by kixelated — COMMENTED** [review]
I think this section could use some more text improvements.

**Pull request merged**